### DISCUSSION TITLE
New Rinkeby Faucet

### DISCUSSION THREAD

**Discussion by PatrickAlphaC** [smartcontractkit/full-blockchain-solidity-course-py#266] [Announcements]
*Big Update*: [New Rinkeby Faucet Located Here](https://faucets.chain.link/rinkeby)

You can find [Backup Faucets here](https://docs.chain.link/docs/link-token-contracts/#rinkeby)

Have fun :)

**Comment by cromewar** [comment]
Incredible new addition!
